DFC.App.FindACourse/DFC.App.FindACourse.csproj

```diff
@@ -13,7 +13,7 @@
     <PackageReference Include="AutoMapper.Extensions.Microsoft.DependencyInjection" Version="7.0.0" />
     <PackageReference Include="DFC.Compui.Cosmos" Version="1.0.61" />
     <PackageReference Include="DFC.Compui.Telemetry" Version="1.0.31" />
-    <PackageReference Include="DFC.Content.Pkg.Netcore" Version="1.0.85" />
+    <PackageReference Include="DFC.Content.Pkg.Netcore" Version="1.0.87" />
```

Comment:
Does it need to be in this project? as it is in the data project - you might be able to remove this nuget reference?

Reply:
Don't think it does, I upgraded it as it was already referencing it, have removed.